### DISCUSSION TITLE
Placeholders

### DISCUSSION THREAD

**Discussion by Notadumdum** [miguelgrinberg/python-socketio#1128] [Q&A] [answered]
![Screenshot_20230206-071108_Chrome](https://user-images.githubusercontent.com/105162586/216896565-486ab491-d307-4404-8ba6-f42abf5d83f5.jpg)
In a situation like this...should events be made to listen to the various placeholders or do creating events for placeholder not matter

**Comment by miguelgrinberg** [comment, marked as answer]
This is part of the Socket.IO protocol. Go to https://github.com/socketio/socket.io-protocol and search for the word "placeholder" if you want to know about it.

**Reply by Notadumdum** [reply]
I checked but I didn't see it being implemented on the client side

**Reply by miguelgrinberg** [reply]
It's an internal thing of the protocol. There's nothing to implement.

**Reply by Notadumdum** [reply]
Considering there are various placeholders with different event names, do I not need to implement a ```socket.on``` for each of them.. is that what you mean sir?

**Reply by miguelgrinberg** [reply]
Please ask the organization that created the server which events you need to implement. I really don't know.
